ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/00_central_germanic.txt

```diff
@@ -1384,6 +1384,20 @@
 			35 = caucasian_brown_hair
 			25 = caucasian_dark_hair
 		}
+		
+		dynasty_of_location_prefix = "dynnp_von"
+
+		# Chance of male children being named after their paternal or maternal grandfather, or their father. Sum must not exceed 100.
+		pat_grf_name_chance = 50
+		mat_grf_name_chance = 5
+		father_name_chance = 10
+		patronym_suffix_male = "dynnpat_suf_son"
```

Comment:
Any basis for this?

Reply:
None at all, see comment above about the Vandal _von/fanē_